packages/strapi-plugin-content-manager/config/routes.json

```diff
@@ -8,6 +8,14 @@
         "policies": []
       }
     },
+    {
+      "method": "GET",
```

Comment:
Since this isn't really some CRUD route, maybe we can add it as an action (`/action/` + POST ?)

Reply:
it is actually a consistent REST route as it represents a specific resource